<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" ignored="0" total="11" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="10804" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:51Z">
    <groups>
    </groups>
    <test name="DashboardPgTest" duration-ms="62" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
      <class name="loginpgtest.LoginPgTest">
        <test-method status="PASS" signature="beforesuite()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforesuite" is-config="true" duration-ms="239" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforesuite -->
      </class> <!-- loginpgtest.LoginPgTest -->
      <class name="dashboardtest.DashboardPgTest">
        <test-method status="FAIL" signature="verirySelePgTitle()[pri:1, instance:dashboardtest.DashboardPgTest@5025a98f]" name="verirySelePgTitle" duration-ms="27" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at dashboardpg.DashboardPg.clickSeleCourse(DashboardPg.java:35)
at dashboardtest.DashboardPgTest.verirySelePgTitle(DashboardPgTest.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verirySelePgTitle -->
        <test-method status="FAIL" signature="veriryJ2eePgTitle()[pri:2, instance:dashboardtest.DashboardPgTest@5025a98f]" name="veriryJ2eePgTitle" duration-ms="3" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at dashboardpg.DashboardPg.clickJ2eeCourse(DashboardPg.java:47)
at dashboardtest.DashboardPgTest.veriryJ2eePgTitle(DashboardPgTest.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veriryJ2eePgTitle -->
        <test-method status="FAIL" signature="veriryPythonPgTitle()[pri:3, instance:dashboardtest.DashboardPgTest@5025a98f]" name="veriryPythonPgTitle" duration-ms="1" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at dashboardpg.DashboardPg.clickPythCourse(DashboardPg.java:58)
at dashboardtest.DashboardPgTest.veriryPythonPgTitle(DashboardPgTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veriryPythonPgTitle -->
        <test-method status="FAIL" signature="veriryPhpPgTitle()[pri:4, instance:dashboardtest.DashboardPgTest@5025a98f]" name="veriryPhpPgTitle" duration-ms="1" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at dashboardpg.DashboardPg.clickPhpCourse(DashboardPg.java:68)
at dashboardtest.DashboardPgTest.veriryPhpPgTitle(DashboardPgTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veriryPhpPgTitle -->
        <test-method status="FAIL" signature="veriryUserPgLoad()[pri:5, instance:dashboardtest.DashboardPgTest@5025a98f]" name="veriryUserPgLoad" duration-ms="1" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:40Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy11.click(Unknown Source)
at dashboardpg.DashboardPg.clickUser(DashboardPg.java:78)
at dashboardtest.DashboardPgTest.veriryUserPgLoad(DashboardPgTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veriryUserPgLoad -->
      </class> <!-- dashboardtest.DashboardPgTest -->
    </test> <!-- DashboardPgTest -->
    <test name="LoginPgTest" duration-ms="10804" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:51Z">
      <class name="loginpgtest.LoginPgTest">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforeMethod" is-config="true" duration-ms="2304" started-at="2022-10-07T18:21:40Z" finished-at="2022-10-07T18:21:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="varifyEmailBySymbol()[pri:1, instance:loginpgtest.LoginPgTest@589838eb]" name="varifyEmailBySymbol" duration-ms="161" started-at="2022-10-07T18:21:42Z" finished-at="2022-10-07T18:21:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- varifyEmailBySymbol -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforeMethod" is-config="true" duration-ms="1378" started-at="2022-10-07T18:21:42Z" finished-at="2022-10-07T18:21:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="varifyEmailByExtn()[pri:2, instance:loginpgtest.LoginPgTest@589838eb]" name="varifyEmailByExtn" duration-ms="163" started-at="2022-10-07T18:21:44Z" finished-at="2022-10-07T18:21:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- varifyEmailByExtn -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforeMethod" is-config="true" duration-ms="1437" started-at="2022-10-07T18:21:44Z" finished-at="2022-10-07T18:21:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="varifyPasswordByUpCase()[pri:3, instance:loginpgtest.LoginPgTest@589838eb]" name="varifyPasswordByUpCase" duration-ms="117" started-at="2022-10-07T18:21:45Z" finished-at="2022-10-07T18:21:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at loginpgtest.LoginPgTest.varifyPasswordByUpCase(LoginPgTest.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- varifyPasswordByUpCase -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforeMethod" is-config="true" duration-ms="1481" started-at="2022-10-07T18:21:45Z" finished-at="2022-10-07T18:21:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="varifyPasswordByLowCase()[pri:4, instance:loginpgtest.LoginPgTest@589838eb]" name="varifyPasswordByLowCase" duration-ms="112" started-at="2022-10-07T18:21:47Z" finished-at="2022-10-07T18:21:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at loginpgtest.LoginPgTest.varifyPasswordByLowCase(LoginPgTest.java:76)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- varifyPasswordByLowCase -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforeMethod" is-config="true" duration-ms="1553" started-at="2022-10-07T18:21:47Z" finished-at="2022-10-07T18:21:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="varifyPasswordByDigits()[pri:5, instance:loginpgtest.LoginPgTest@589838eb]" name="varifyPasswordByDigits" duration-ms="114" started-at="2022-10-07T18:21:49Z" finished-at="2022-10-07T18:21:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- varifyPasswordByDigits -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:loginpgtest.LoginPgTest@589838eb]" name="beforeMethod" is-config="true" duration-ms="1542" started-at="2022-10-07T18:21:49Z" finished-at="2022-10-07T18:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="varifyNewPgLoad()[pri:6, instance:loginpgtest.LoginPgTest@589838eb]" name="varifyNewPgLoad" duration-ms="412" started-at="2022-10-07T18:21:50Z" finished-at="2022-10-07T18:21:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- varifyNewPgLoad -->
      </class> <!-- loginpgtest.LoginPgTest -->
    </test> <!-- LoginPgTest -->
  </suite> <!-- Suite -->
</testng-results>
